ui/src/pages/admin_page/UsersAdmin.tsx

```diff
@@ -217,30 +216,21 @@ function DeleteUserModal(props: DeleteUserModalProps): JSXElement {
         </Show>
 
         <div class="modal-action">
-          <button
-            class={clsx(
-              'btn btn-outline',
-              deleteForm.submitting && 'btn-disabled'
-            )}
+          <Button
+            label={t('cancel')}
+            class="btn-outline"
+            isLoading={deleteForm.submitting}
             onClick={(e) => {
               e.preventDefault()
               props.onClose()
             }}
-          >
-            {t('cancel')}
-          </button>
-          <button
-            class={clsx(
-              'btn btn-error',
-              deleteForm.submitting && 'btn_disabled'
-            )}
-            type="submit"
-          >
-            {t('delete')}
-            <Show when={deleteForm.submitting}>
-              <span class="loading loading-ball" />
-            </Show>
-          </button>
+          />
+
+          <Button
+            label={t('delete')}
+            color="error"
+            isLoading={deleteForm.submitting}
+          />
```

Comment:
Not sure if it is because of these changes, but the delete button does not work anymore in the admin